config/locales/en.yml

```diff
@@ -399,6 +399,7 @@ en:
           collection: Save and Exit
           work_version: Save as Draft & Exit
           admin_save: Save and Exit
+        request_curation: Request Curation & Save
```

Comment:
Save as Draft is definitely more descriptive & I think it would be a good change. I think it might be good to leave Request Curation first though. It's super minor, but I think having it first will help catch users attention more & make sure someone doesn't click it without reading the whole thing thinking they're just saving it.

Reply:
Good point - sounds good to me!